test/conformance/testing/include/uur/fixtures.h

```diff
@@ -226,11 +230,12 @@ struct urMemImageTest : urContextTest {
         /*.height =*/16,
         /*.depth =*/1,
         /*.arraySize =*/1,
-        /*.rowPitch =*/16 * sizeof(char[4]),
-        /*.slicePitch =*/16 * 16 * sizeof(char[4]),
+        /*.rowPitch =*/0,   //16 * sizeof(char[4]),
```

Comment:
Why is this commented out?

Reply:
Nice spot, yeah that's a mistake during debugging. Slipped my todo list when cleaning up this branch. Will also remove the commented out test in urMemBufferCreateWithNativeHandle.cpp

Reply:
Updated this, according [to the spec](https://oneapi-src.github.io/unified-runtime/core/api.html#urmemimagecreate) rowPitch and slicePitch should be 0